backend/src/controllers/CMSTutorialController.ts

```diff
@@ -41,8 +41,8 @@ export default class CMSTutorialController {
             const { name, html, markdown, category } = req.body;
             const dto = new Tutorial(name, html, markdown, category, userId, username, true);
 
-            TutorialUpdateService.createDocument<Tutorial>("tutorials", dto).then(
-                () => res.status(201).send({ ok: 1, n: 1 }).end(),
+            DbUpdateService.createDocument<Tutorial>("tutorials", dto).then(
+                (result) => res.status(201).send(result).end(),
                 () => res.status(500).send(INTERNAL_ERROR_TEXT + "TODO").end()
```

Comment:
Have you updated the comments with todo as well?

Reply:
No no, the "TODO" is a diff ticket, the result sent here is proper evaluation of the result + testing is more consistent with a result directly from the service we're using, I initially hardcoded it lol.